src/test/java/util/DummyDataGenerator.java

```diff
@@ -24,6 +24,9 @@
 
 public class DummyDataGenerator {
```

Comment:
- Since this class is under the tests I would remove the Dummy keywords, example name the class as DataGenerator since we only use it in tests it is obvious that the values are made up and are dummy.

Reply:
Agreed